scripts/releaseChangelog.mjs

```diff
@@ -0,0 +1,34 @@
+import fetch from "node-fetch";
```

Comment:
we dont need to import fetch, i think its part of nodejs builtin globals

Reply:
addressed